oak_functions/loader/benches/lookup.rs

```diff
@@ -33,125 +31,199 @@ use oak_functions_loader::{
 use prost::Message;
 use proto::{benchmark_request::Action, BenchmarkRequest, LookupTest};
 use rand::SeedableRng;
-use std::{collections::HashMap, sync::Arc, time::Duration};
-use test::Bencher;
+use std::{collections::HashMap, sync::Arc};
+use tokio::runtime::Runtime;
 
 const MANIFEST_PATH: &str = "examples/benchmark/module/Cargo.toml";
+const SINGLE_REQUEST_ITERATIONS: u32 = 1;
+const MULTI_REQUEST_ITERATIONS: u32 = 101;
```

Comment:
Why 101? I expected 100.

Reply:
101 is so that it is easier to divide mentally. Doing 101 iterations and subtracting a single iteration leaves the approximate duration of 100 iterations.